packages/directlinespeech/src/createCustomEvent.js

```diff
@@ -0,0 +1,19 @@
+export default function createCustomEvent(name, eventInitDict = {}) {
+  if (name === 'error') {
+    if (typeof ErrorEvent === 'function') {
+      return new ErrorEvent(name, eventInitDict);
+    }
+  } else if (typeof CustomEvent === 'function') {
+    return new CustomEvent(name, eventInitDict);
+  }
+
+  const event = document.createEvent('Event');
+
+  event.initEvent(name, true, true);
+
+  Object.entries(eventInitDict).forEach(([name, value]) => {
+    event[name] = value;
+  });
```

Comment:
Prefer to keep it this way because, in future, when we call `createCustomEvent('something', { detail: 'Hello' })`:

- Modern browsers: `new CustomEvent('something', { detail: 'Hello' })`
- IE11: `document.createEvent()` + `event.initEvent('something')` only, and we lose the `{ detail: 'Hello' }`

Then we are hiding a potential bug.

We can fix it by:

- Implement today and left it unused
- When someone send `eventInitDict`, throw "not implemented" exception

I am adding a test to make sure this unused code is covered.